clusters/l3-sqnc/monitoring/values.yaml

```diff
@@ -0,0 +1,79 @@
+alertmanager:
+  enabled: true
+  serviceMonitor:
+    selfMonitor: true
+    namespace: monitoring
+  config:
+    receivers:
+      - name: "null"
+      # - 
+      # - name: slack
+      #   webhook_configs:
+      #     - send_resolved: true
+      #       url: <secret>
+prometheus:
+  serviceMonitor:
+    selfMonitor: true
+    namespace: monitoring
+  prometheusSpec:
+    retention: 24h
```

Comment:
The retention is way too long for production.  @mattdean-digicatapult thoughts on how long we want to keep stuff?